test/verify/check-pages

```diff
@@ -186,13 +186,16 @@ OnCalendar=daily
 
         # Lets try changing the language
 
+        b.go("/system")
+        b.enter_page("/system")
+        b.switch_to_top()
```

Comment:
That is really weird -- can you please mark this as `# HACK`? its really not obvious why this happens. open_lang_modal() already does a `switch_to_top()`, and that is an entirely trivial operation (just setting an internal variable), so that line should go. More importantly, it's not clear why the language switcher would work from one frame but not the other one. Is that some "mouse click gets blocked by some element" situation again? how does it look like?

Reply:
Good point about `switch_to_top`, I'll remove that.

The problem was doing the `go("/system") after changing the language. It sometimes didn't have any effect. Changing the language always works.

I can debug this further, but I could never reproduce it locally, so that's going to be a bit of a pain...

Reply:
(Ahh, I think I know. The router starts working a bit later since the shell rewrite, because I though there can be no messages before we create iframes. But tests actually send messages to implement `b.go`, and if they come to early after a reload, we'll miss them. So let's wait for the shell to be fully loaded again before navigating.)

Reply:
Success: https://cockpit-logs.us-east-1.linodeobjects.com/pull-21061-e8608f32-20241016-095240-fedora-40-firefox-expensive/log.html